### DISCUSSION TITLE
onFieldReact的一些疑问

### DISCUSSION THREAD

**Discussion by henryzp** [alibaba/formily#3197] [Q&A 2.x]
先说问题：该事件能不能做到第一次不触发？

场景是这样的：体积 = 长 * 宽 * 高，即体积依赖长、宽、高三个字段

但是在保存前，体积可以手动修改成其他值。。那么这时候保存之后，会重新计算体积，是有问题的。。用户希望的还是他自己自定义的体积。

当然我可以去监听长、宽、高这三个字段，然后做一些处理，就是想问一下，有没有更优雅的写法？

**Comment by wweggplant** [comment]
监听保存的动作是否可以满足？
